Makefile

```diff
@@ -58,8 +58,8 @@ flatpak-install: translations-to-qm
 
 lint:
 	flake8
-	isort --check-only .
 	black --check .
+	ruff check . --fix
```

Comment:
Why `--fix`? This is intended as a check only. That's why black is run with `--check`.